### PR TITLE
added artifacs to archive for kie-wb-common

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: job-dsls/jobs/new_pr_jobs.groovy [hunk 1]
```diff
@@ -81,7 +81,11 @@ def final REPO_CONFIGS = [
         "kie-wb-common"             : [
                 timeoutMins: 300,
                 label: "kie-rhel7 && kie-mem16g && gui-testing",
-                executionNumber : 25
+                executionNumber : 25,
+                artifactsToArchive: [
+                        "kie-wb-common-stunner-bpmn-kogito-runtime/target/screenshots/**",
+                        "kie-wb-common-dmn-webapp-kogito-runtime/target/screenshots/**"
```
Comment (mareknovotny) [inline job-dsls/jobs/new_pr_jobs.groovy:87]: these modules are not on top level of the repository, maybe do just wild card here 
```suggestion
                        "**/target/screenshots/**"
```

**Review by Ginxo — APPROVED** [review]
LGTM

File: job-dsls/jobs/new_pr_jobs.groovy (hunk 1 above)
Comment (mbiarnes) [reply job-dsls/jobs/new_pr_jobs.groovy]: done

Comment (mbiarnes) [reply job-dsls/jobs/new_pr_jobs.groovy]: feel free to merge

**Comment by mbiarnes** [comment]
Jenkins retest this

**Review by jomarko — APPROVED** [review]
Any regular expression we use, it needs to match https://github.com/kiegroup/kie-wb-common/pull/3405#issuecomment-684684150, I have no way to try this change manually so approving just as seems reasonable.

**Comment by mbiarnes** [comment]
@jomarko this change was done "online" by @Ginxo .
When  https://github.com/kiegroup/kie-wb-common/pull/3405 finishes to build (the normal pull request) the screenshot should be there.

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**